### PR TITLE
feat: Support soft-wrapping in SqlEditor for table calculations

### PR DESCRIPTION
**Pull request by filp**
### Description:
Adds support for soft-wrapping while editing table calculation expressions.

Kapture 2024-01-30 at 14 33 30

Simply exposes the underlying ace editor feature, and should be straight-forward to include this behavior in an SqlRunner component follow-up.

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Review by joaoviana — APPROVED** [review]
looks great!

File: packages/frontend/src/features/tableCalculation/components/SqlForm.tsx [hunk 1]
```diff
@@ -41,8 +48,42 @@ const SqlEditor = styled(AceEditor)<
...
+                isSoftWrapEnabled
+                    ? 'Disable editor soft-wrapping'
+                    : 'Enable editor soft-wrapping'
+            }
+            withArrow
+            position="left"
+        >
+            <ActionIcon onClick={onToggleSoftWrap}>
+                {isSoftWrapEnabled ? (
+                    <IconTextWrapDisabled size="1rem" />
```
Comment (joaoviana) [inline packages/frontend/src/features/tableCalculation/components/SqlForm.tsx:75]: non -blocking, but we usually use `MantineIcon` component for `icons`, like so: 
```tsx
<MantineIcon icon={IconTextWrapDisabled} />
```

File: packages/frontend/src/features/tableCalculation/components/SqlForm.tsx [hunk 2]
```diff
@@ -41,8 +48,42 @@ const SqlEditor = styled(AceEditor)<
...
+    onToggleSoftWrap: () => void;
+}> = ({ isSoftWrapEnabled, onToggleSoftWrap }) => (
+    <div
+        style={{
+            position: 'absolute',
+            bottom: 0,
+            right: 0,
+            // Avoids potential collision with ScrollArea scrollbar:
+            marginRight: '5px',
+        }}
```
Comment (joaoviana) [inline packages/frontend/src/features/tableCalculation/components/SqlForm.tsx:62]: another non-blocking, but you can use Mantine's `Box` and pass these styles as props

**Review by joaoviana — CHANGES_REQUESTED** [review]
moving to requested so we keep using `MantineIcon` and enable global changes to the icon component in case we want to change the default size etc 👍

**Comment by filp** [comment]
Thanks @joaoviana! Made a quick pass based on your feedback.

**Review by joaoviana — APPROVED** [review]
🚢 it!

**Pull request merged**